Error compiling jellyfin

I'm trying to compile jellyfin according to https://jellyfin.org/docs/general/administration/building.html

I get this error at step 8/12:

```
$ ./build ubuntu-package-x64 all
~/jellyfin/deployment ~/jellyfin
> Processing platform ubuntu-package-x64
~/jellyfin/deployment/ubuntu-package-x64 ~/jellyfin/deployment ~/jellyfin
>> Processing action build
>> Processing action package
Sending build context to Docker daemon  9.032MB
Step 1/12 : FROM microsoft/dotnet:2.2-sdk-bionic
2.2-sdk-bionic: Pulling from microsoft/dotnet
7ddbc47eeb70: Pull complete
c1bbdc448b72: Pull complete
8c3b70e39044: Pull complete
45d437916d57: Pull complete
750e6c6007ed: Pull complete
c1a7f7e493d0: Pull complete
40671ea77a34: Pull complete
27252d8f48c5: Pull complete
8c9b8374df2a: Pull complete
86cd289bed56: Pull complete
Digest: sha256:9d30f00651ffe892c93ecd8c5351eea65785aae44c13011ae49a874be7ab61d1
Status: Downloaded newer image for microsoft/dotnet:2.2-sdk-bionic
 ---> 746208285f16
Step 2/12 : ARG SOURCE_DIR=/jellyfin
 ---> Running in 015c6cb0bdec
Removing intermediate container 015c6cb0bdec
 ---> 9922663f91b4
Step 3/12 : ARG PLATFORM_DIR=/jellyfin/deployment/ubuntu-package-x64
 ---> Running in d120c6379e54
Removing intermediate container d120c6379e54
 ---> 9507bda84c30
Step 4/12 : ARG ARTIFACT_DIR=/dist
 ---> Running in 85c3b7a218cf
Removing intermediate container 85c3b7a218cf
 ---> 8118d1d26d58
Step 5/12 : ENV SOURCE_DIR=/jellyfin
 ---> Running in 91ccfacf9d50
Removing intermediate container 91ccfacf9d50
 ---> 7c8618c17790
Step 6/12 : ENV ARTIFACT_DIR=/dist
 ---> Running in d055bdc7d3b8
Removing intermediate container d055bdc7d3b8
 ---> 17d7b80e39c9
Step 7/12 : ENV DEB_BUILD_OPTIONS=noddebs
 ---> Running in 0d69d49f1375
Removing intermediate container 0d69d49f1375
 ---> 56ee69a3207b
Step 8/12 : RUN apt-get update  && apt-get install -y apt-transport-https debhelper gnupg wget devscripts mmv libc6-dev libcurl4-openssl-dev libfontconfig1-dev libfreetype6-dev libssl-dev  && ln -sf ${PLATFORM_DIR}/docker-build.sh /docker-build.sh  && mkdir -p ${SOURCE_DIR} && ln -sf ${PLATFORM_DIR}/pkg-src ${SOURCE_DIR}/debian
 ---> Running in a86d8c8e7552
OCI runtime create failed: container_linux.go:345: starting container process caused "process_linux.go:430: container init caused \"rootfs_linux.go:58: mounting \\\"proc\\\" to rootfs \\\"/var/lib/docker/vfs/dir/27850e378dcbcf94470a57817f874e4310b4c346db5d035bd259932ad2b1ff31\\\" at \\\"/proc\\\" caused \\\"permission denied\\\"\"": unknown
Unable to find image 'jellyfin-ubuntu-build:latest' locally
docker: Error response from daemon: pull access denied for jellyfin-ubuntu-build, repository does not exist or may require 'docker login'.
See 'docker run --help'.
mv: cannot stat '/home/user/jellyfin/deployment/ubuntu-package-x64/pkg-dist-tmp/deb/*': No such file or directory
>> Processing action clean
Error: No such image: jellyfin-ubuntu-build
```

**System (please complete the following information):**
 - OS: Ubuntu 18.04
 - Browser: N/A
 - Jellyfin Version: latest from github
 - Reverse proxy: no

I have no experience with docker.  I just installed it with `apt install docker.io` on my system and then ran the jellyfin build script.  What can I do to fix this?

I added my user to docker group and the build gets further now, all the way to step 12/12, but still errors out.  

There are a few errors along the way but this one seems to be the main one:
```
/usr/share/dotnet/sdk/2.2.207/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(150,5): error NETSDK1045: The current .NET SDK does not support targeting .NET Standard 2.1.  Either target .NET Standard 2.0 or lower, or use a version of the .NET SDK that supports .NET Standard 2.1. [/jellyfin/MediaBrowser.WebDashboard/MediaBrowser.WebDashboard.csproj]
/usr/share/dotnet/sdk/2.2.207/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(137,5): error NETSDK1045: The current .NET SDK does not support targeting .NET Core 3.0.  Either target .NET Core 2.2 or lower, or use a version of the .NET SDK that supports .NET Core 3.0. [/jellyfin/Jellyfin.Server/Jellyfin.Server.csproj]
debian/rules:44: recipe for target 'override_dh_auto_build' failed
make[1]: Leaving directory '/jellyfin'
make[1]: *** [override_dh_auto_build] Error 1
debian/rules:35: recipe for target 'binary' failed
make: *** [binary] Error 2
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
mv: cannot stat '/home/user/jellyfin/deployment/ubuntu-package-x64/pkg-dist-tmp/deb/*': No such file or directory
```

I have the full build output at https://pastebin.com/raw/qMtAhfQG
Is the build happening in a docker container or my host?
Thanks for your help.

--- a/deployment/ubuntu-package-x64/Dockerfile
+++ b/deployment/ubuntu-package-x64/Dockerfile
@@ -1,4 +1,4 @@
-FROM microsoft/dotnet:2.2-sdk-bionic
+FROM microsoft/dotnet:3.0-sdk-bionic
 # Docker build arguments
 ARG SOURCE_DIR=/jellyfin
 ARG PLATFORM_DIR=/jellyfin/deployment/ubuntu-package-x64

